tests/functional-ui/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "targets": {
+    "lint": {},
+    "check": {},
+    "functional-ui": {
+      "cache": false,
```

Comment:
Why can this not be cached?

Reply:
So what should I review if we do not run it? Are we trying to merge code that is not actually used?